api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -506,6 +507,38 @@ message HttpConnectionManager {
   //
   // 3. Tracing decision (sampled, forced, etc) is set in 14th byte of the UUID.
   RequestIDExtension request_id_extension = 36;
+
+  // Configuration of local reply returned by Envoy. Allows to specify mappings and format of
+  // response.
+  LocalReplyConfig local_reply_config = 38;
+}
+
+message LocalReplyConfig {
+  // Configuration of list of mappers which allows to filter and change local response.
+  // The client will iterate through mappers until first match.
+  repeated ResponseMapper mapper = 1;
+
+  // Allows to define custom format of local reply.
+  // It is allowed to use :ref:`command operators <config_access_log_command_operators>`
+  // which will be resolved to actual data.
```

Comment:
What is the behavior if absent? Should this be required if a local reply config is specified?

Reply:
No, it is optional.  LocalReplyConfig may have some mappers to modify status_code and body.  But it may NOT need to have format.  If format is not specified,  the boxy will be written as plain text to respnose.